src/compounds/ActionsGroup/actions/ViewFiles.jsx

```diff
@@ -108,7 +108,7 @@ const ViewFiles = ({ backgroundColor, files, handleClose, onSubmit }) => {
         <Tabs defaultActiveKey='files' id='document-tabs' justify fill>
           {documentTabs && documentTabs.map((tab) => {
             const { eventKey, title, status } = tab
-            const filteredFiles = files.filter((f) => (status === f.status) || (status === 'Other File' && f.status === null))
+            const filteredFiles = files ? (files.filter((f) => (status === f.status) || (status === 'Other File' && f.status === null))) : ([])
```

Comment:
according to the prop type definition for `files` in this component below, shouldn't it already at minimum be an empty array? if not, then the prop type validation should be failing. in which case the fix would be to adjust the value that we're in passing in for `files`.

if `files = []` or `files = [{}]`, the filter method would return an empty array.

``` js
files: PropTypes.arrayOf(
    PropTypes.shape({ ... }),
  ).isRequired
```

Reply:
I just came across this locally while working on some other stuff. the problem is that `files` is undefined while in loading state. passing undefined into this component complains as expected.

imo there are 2 options:
1. make sure we never pass in an undefined value
2. make this component accept an undefined value.

I prefer option 1 because as the error you're looking at points out...we can't call `filter` on an undefined value. if you're fine with that, can you approve https://github.com/scientist-softserv/webstore/pull/225? I created it to fix my local problems and then realized it solved this problem.

Reply:
sounds great!

Reply:
I just approved the always define files